start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
```

Comment:
This may be a bit of "bash golf" on my part, but you could do

```bash
which triton > /dev/null
if [ $? -ne 0 ]; then
    echo "hey no triton!"
fi
```